### ISSUE TITLE
Base |> pipe operator crashing install

### ISSUE THREAD

**Issue by tcash21** [cole-johanson/smartsheetr#6]
Hello, 

This package is going to be incredibly useful for myself and a client. However their R version on RConnect is only 4.0.5 and does not support the `|>` pipe yet. 

I don't know if it's a major pain to support `%>%` for now instead, and realize it may be a lot to ask, I'll cross my fingers that they upgrade their R version to at least 4.1. 

Thank you either way for this package!

P.S. Happy to make this change if it's helpful and issue a PR, but wanted to ask you first.

```
devtools::install_github("cole-johanson/smartsheetr")
.
.
.
105: ss_resp_data_to_dataframe <- function(resp_data) {
106:   purrr::map(resp_data, as.data.frame) |>
                                             ^
ERROR: unable to collate and parse R files for package ‘smartsheetr’
```

**Comment by cole-johanson** [comment]
Hello! I'm so glad this package is useful! I would like to keep this package as lean as possible, and R 4.1.0 was released 2.5 years ago, so I think the need for the magrittr pipe will continue to diminish over time. 

My suggestion would be to:
1. Fork/Pull the package, and create a branch with the `|>`s replaced with `%>%`s (using search/replace all). 
2. Run `usethis::use_import_from('magrittr','%>%')`
3. Run tests
4. Push to your own repo
5. Install locally with `remotes::install_local()`, or install from your own repo using `remotes::install_github()`

Let me know if that workflow won't work for you for whatever reason and I will consider adding a branch for you.

**Comment by tcash21** [comment]
They updated their RConnect so we are good to go, but I appreciate the workaround advice. Thank you again!
